src/main/java/org/phyloref/jphyloref/commands/WebserverCommand.java

```diff
@@ -343,11 +351,20 @@ public void handleStatement(org.eclipse.rdf4j.model.Statement st) throws RDFHand
               type.asOWLClass().getIRI().equals(PhylorefHelper.IRI_CDAO_NODE)
           ))
           .map(indiv -> indiv.getIRI().toString())
+          // Strip the default prefix on the node URI if present.
+          .map(iri -> !iri.startsWith(DEFAULT_URI_PREFIX) ? iri : iri.substring(DEFAULT_URI_PREFIX.length()))
```

Comment:
Any reason you wouldn't just use [String.replace()?](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace)?
```suggestion
          .map(iri -> iri.replace(new RegExp("^" + DEFAULT_URI_PREFIX), ''))
```